graph_node_metrics.py

```diff
@@ -312,9 +312,34 @@ def collect_graph_networks_metrics(client):
         logger.error(f"Error collecting graph networks: {e}")
         raise
 
+def collect_active_peers_count(client):
+    try:
+        query = gql('''
+        query {
+            active_peers:peers(where: {and: [{currentCapacityCommitment_:{status:Active}}, {provider_: {approved: true}}]}) {
+                computeUnitsTotal
+            },
+            peers_total:peers(where: {and: [{provider_: {approved: true}}]}) {
+                computeUnitsTotal
+            }
+        }
```

Comment:
You probably can do just one `peers()`, and ask for CC status instead of `computeUnitsTotal`.

Then you can count Active in Python. This way, it would be less load on the Indexer, and less data over the wire.